templates/zerver/integrations/github-actions.md

```diff
@@ -1,8 +1,25 @@
-It's easy to send Zulip messages from GitHub Actions! This is useful:
+# Zulip GitHub Actions integration
 
-  * If you want to report the success or failure of an Action to Zulip.
-  * If you want to send output from your actions to Zulip.
+It's easy to send Zulip messages from GitHub Actions!
 
-Instructions are available [here](https://github.com/zulip/github-actions-zulip#readme).
+{start_tabs}
 
-See also the [GitHub integration](/integrations/doc/github).
+1. {!create-a-generic-bot.md!}
+
+1. To send Zulip notifications whenever your workflow runs, add the
```

Comment:
This was previously in 2 separate instructions, that has now been clubbed together, as they don't make much sense separately.
We introduce the action that needs to be added, and the values with which it needs to be configured in the same instruction.